src/services/ubi/community.ts

```diff
@@ -643,14 +653,30 @@ export default class CommunityService {
             ],
             where: {
                 communityId: community.publicId,
-                active: true,
+                ...activeCondition,
             },
         });
+
+        const address = result.map((el) => `'${el.address}'`);
+        const beneficiariesAdded:
+            | { count: number; from: string }[]
+            | undefined = await this.sequelize.query(
+            `SELECT count("address"), "from" FROM ubi_beneficiary_registry WHERE "from" IN (${address.toString()}) GROUP BY "from"`,
+            {
+                raw: true,
+                type: QueryTypes.SELECT,
+            }
+        );
+
         return result.map((r) => {
             const manager = r.toJSON() as ManagerAttributes;
+            const addedBeneficiaries = beneficiariesAdded.find(
+                (el) => el.from === manager.address
+            );
             return {
                 ...manager,
                 isDeleted: !manager.user,
+                addedBeneficiaries: addedBeneficiaries ? Number(addedBeneficiaries?.count) : 0,
```

Comment:
you can use https://www.postgresqltutorial.com/postgresql-coalesce/ on the query to avoid nulls :wink:

Reply:
in this case `coalesce` didn't help because if a manager didn't have registries in `ubi_beneficiary_registry` he won't be returned in the query. But I changed this line, I think it's simpler